compiler/src/dmd/backend/cgxmm.d

```diff
@@ -1863,4 +1871,35 @@ void cloadxmm(ref CodeBuilder cdb, elem *e, regm_t *pretregs)
     cload87(cdb, e, pretregs);
 }
 
+/***********************************
+ * Determine if we can load a constant into an XMM register
+ * with instructions.
+ * Params:
+ *      e = constant
+ * Returns:
+ *      true if it can be done
+ */
+@trusted
+bool loadxmmconst(elem *e)
+{
+    //printf("loadxmmconst() "); elem_print_const(e); printf("\n");
+    const sz = tysize(e.Ety);
+    ubyte* p = cast(ubyte*)&e.EV;
+    assert(sz >= 1);
+
+    if (sz > 8)
+        return false;
+
+    // true only if all ones or all zeros
+    const b = p[0];
+    if (b != 0 /*&& b != 0xFF*/)
```

Comment:
The comment says "true only if all ones or all zeros" but the `0xFF` check is commented out.

Reply:
That's because adding the -1 support requires more work, so I thought I'd just start with the 0 support.